view/frontend/templates/js/replacejs.phtml

```diff
@@ -694,7 +694,7 @@ ob_start();
             }
             params = params.join('&');
 
-            var BC = BoltCheckout.configure(new Promise(function (resolve, reject) {
+            let cartPromise = new Promise(function (resolve, reject) {
```

Comment:
Hmm I'm not sure if I understand motivation here. IMO The original code was what we want - we pass promise to `BoltCheckout.configure` so Bolt can properly wait for Promise even though user clicks checkout button before data is ready.

After change `BoltCheckout.configure` won't be called after Promise is resolved, if user clicks too early they would see error because Bolt was never initialized

Reply:
@danac-gs raised the same concern in the Asana ticket - he proposed a fix and I'll let him tackle it. Closing this ticket.